README.md

```diff
@@ -49,7 +49,66 @@ devtools::install_github("UBC-MDS/feature-selection-r")
 
 ## Usage
 
-- TODO
+To guide you with an example of how to use this package, we will use the [Friedman dataset](https://www.rdocumentation.org/packages/tgp/versions/2.4-14/topics/friedman.1.data).
+It is important to mention that the forward_selection, recursive_feature_elimination, and simulated_annealing functions need a custom scorer provided by the user, that returns the name of the column with the lowest coefficient weight.
```

Comment:
Don't put this here. Just add a small comment like I suggested at the top of each scorer and make sure the comments reflect what is actually being returned.

The important one is the `recursive_feature_elimination` function because it is not very clear what the code is doing. For the other scorers, it's clear that it is returning MSE.